src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -236,7 +236,7 @@ class PerformanceLanding extends React.Component<Props, State> {
           conditions.getTagValues('transaction.duration')
         );
       } else {
-        modifiedConditions.setTagValues('transaction.duration', ['>0', '<60min']);
+        modifiedConditions.setTagValues('transaction.duration', ['>=0', '<=60min']);
```

Comment:
Correct me if I'm wrong, but wasn't `>0` intentional in order to filter out 0 duration transactions?